projects/kit/components/input-date/multi-date/input-multi-date.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {Directive} from '@angular/core';
+import {TuiDay} from '@taiga-ui/cdk';
+import {AbstractTuiTextfieldHost, tuiAsTextfieldHost} from '@taiga-ui/core';
+
+import {tuiImmutableUpdateMultiDates} from './immutable-update';
+import {TuiInputMultiDateComponent} from './input-multi-date.component';
+
+@Directive({
+    selector: 'tui-input-date[multiple]',
+    providers: [tuiAsTextfieldHost(TuiInputMultiDateDirective)],
+})
+export class TuiInputMultiDateDirective extends AbstractTuiTextfieldHost<TuiInputMultiDateComponent> {
+    override get value(): string {
+        return this.host.value.toString();
```

Comment:
nit: `String(this.host.value)` is a bit shorter :)

Reply:
fixed